srcpkgs/dendrite/template

```diff
@@ -0,0 +1,38 @@
+# Template file for 'dendrite'
+pkgname=dendrite
+version=0.6.3
+revision=1
+build_style=go
+go_import_path="github.com/matrix-org/dendrite"
+go_package="./cmd/create-account ./cmd/dendrite-monolith-server
+ ./cmd/generate-config ./cmd/generate-keys"
+short_desc="Dendrite is a second-generation Matrix homeserver written in Go"
+maintainer="Joel Beckmeyer <joel@beckmeyer.us>"
+license="Apache-2.0"
+homepage="https://matrix.org/docs/projects/server/dendrite"
+distfiles="https://github.com/matrix-org/dendrite/archive/v${version}.tar.gz"
+checksum=0c6d90448f3619ed4a4c56acdf8e674473e24521ac8135db87a7d88f03755b39
+
+system_accounts="_dendrite"
+_dendrite_homedir="/var/lib/dendrite"
+
+make_dirs="
+	/var/lib/dendrite 0700 _dendrite _dendrite
+	/var/log/dendrite 0755 _dendrite _dendrite
+	/etc/dendrite 0755 _dendrite _dendrite"
+
+pre_configure() {
+	vsed -e 's,\./logs,/var/log/dendrite,' \
+	 -e 's,: matrix_key.pem,: /etc/dendrite/matrix_key.pem,' \
+	 -i dendrite-config.yaml
+}
+
+post_install() {
+	vsconf dendrite-config.yaml dendrite.yaml
+	vsv dendrite-monolith-server
+
+	cd "${DESTDIR}"/usr/bin
+	for f in *; do
+		mv {,dendrite-}"${f}"
+	done
```

Comment:
Doesn't this add a `dendrite-` prefix to all files, including the ones that already have this prefix?